docs-v2/reference/cli.mdx

```diff
@@ -13,7 +13,19 @@ npm install nango -g
 
 In the folder where you want your integration folder (e.g. root of your project), run:
 ```bash
-nango init # Creates `./nango-integrations` with initial config
+nango init
+```
+
+This creates the `./nango-integrations` folder with some initial configuration and an example
+sync script. The `nango-integrations` directory looks like this:
+```
+nango-integrations/
+├── .env
+├── nango.yaml
+├── models.ts
+└── demo-github-integration
```

Comment:
I would indicate with a comment that this is the integration ID, and must match an entry in the `nango.yaml` 
 and UI.